src/open_producten/fixtures/default_admin_index.json

```
@@ -1,293 +1,253 @@
 [
-{
-    "model": "admin_index.appgroup",
-    "fields": {
-        "order": 0,
-        "translations": {
-            "nl": "Producten"
-        },
-        "name": "Products",
-        "slug": "products",
-        "models": [
-            [
-                "products",
-                "data"
-            ],
-            [
-                "products",
-                "product"
+    {
+        "model": "admin_index.appgroup",
+        "fields": {
+            "order": 0,
+            "translations": {
+                "en": "Products"
+            },
+            "name": "Producten",
+            "slug": "Producten",
+            "models": [
+                [
+                    "producten",
+                    "product"
+                ]
             ]
-        ]
-    }
-},
-{
-    "model": "admin_index.appgroup",
-    "fields": {
-        "order": 1,
-        "translations": {
-            "nl": "Product typen"
-        },
-        "name": "Product types",
-        "slug": "product-types",
-        "models": [
-            [
-                "producttypes",
-                "category"
-            ],
-            [
-                "producttypes",
-                "categoryproducttype"
-            ],
-            [
-                "producttypes",
-                "condition"
-            ],
-            [
-                "producttypes",
-                "field"
-            ],
-            [
-                "producttypes",
-                "file"
-            ],
-            [
-                "producttypes",
-                "link"
-            ],
-            [
-                "producttypes",
-                "price"
-            ],
-            [
-                "producttypes",
-                "priceoption"
-            ],
-            [
-                "producttypes",
-                "producttype"
-            ],
-            [
-                "producttypes",
-                "question"
-            ],
-            [
-                "producttypes",
-                "tag"
-            ],
-            [
-                "producttypes",
-                "tagtype"
-            ],
-            [
-                "producttypes",
-                "uniformproductname"
+        }
+    },
+    {
+        "model": "admin_index.appgroup",
+        "fields": {
+            "order": 1,
+            "translations": {
+                "en": "Product types"
+            },
+            "name": "Product typen",
+            "slug": "product-typen",
+            "models": [
+                [
+                    "producttypen",
+                    "onderwerp"
```

```suggestion
                    "thema"
```
IRC, this was renamed to thema? Regardless it does not exist